### PR TITLE
discard export PYTHONPATH manually and fix east label map dynamic shape

### PR DESCRIPTION
**Pull request by LDOUBLEV**
- discard export PYTHONPATH manually
- fix issue #206

### CODE REVIEW

File: tools/eval.py [hunk 1]
```diff
@@ -17,6 +17,11 @@
 from __future__ import print_function
 
 import os
+import os
```
Comment (tink2123) [inline tools/eval.py:20]: 多了一行？

File: tools/infer/predict_rec.py [hunk 2]
```diff
@@ -11,6 +11,11 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
+import sys
+__dir__ = os.path.dirname(__file__)
+sys.path.append(__dir__)
+sys.path.append(os.path.join(__dir__, '../..'))
 
 import utility
```
Comment (tink2123) [inline tools/infer/predict_rec.py:20]: import tools.infer.utility as utility ？

File: tools/eval.py (hunk 1 above)
Comment (LDOUBLEV) [reply tools/eval.py:20]: Done !

File: tools/infer/predict_rec.py (hunk 2 above)
Comment (LDOUBLEV) [reply tools/infer/predict_rec.py:20]: Done !

**Pull request merged**